greenpithumb/dht11.py

```diff
@@ -0,0 +1,50 @@
+import datetime
+import threading
+
+from Adafruit_DHT import common
+
+_FRESHNESS_THRESHOLD = 2
+
+
+class CachingDHT11(object):
+    """Caching DHT11 sensor wrapper.
```

Comment:
Suggest adding as much information as can fit on the first line, so like, "Wrapper around a DHT11 that caches sensor readings."